custom/icds_reports/templates/icds_reports/icds_app/month-filter.html

```diff
@@ -1,11 +1,11 @@
 <script type="text/ng-template" id="monthModalContent.html">
     <div class="modal-header">
-        <h3 class="modal-title" id="modal-title">Select Month and Year<i style="float: right;" ng-click="$ctrl.close()" class="fa fa-close pointer"></i></h3>
+        <h3 class="modal-title" id="modal-title">Select Month and Year<i style="float: right;" ng-show="!$ctrl.showMessage" ng-click="$ctrl.close()" class="fa fa-close pointer"></i></h3>
     </div>
     <div class="modal-body" id="modal-body">
           <div class="alert alert-info" ng-show="$ctrl.showMessage">
-      <strong>Info!</strong>  Supported period is from Feb’19.
-    </div>
+            Service Delivery Dashboard is available from February 2019
```

Comment:
Nit: I think it would be more clear if the text read: "Service Delivery Dashboard is only available from February 2019 onwards."

Reply:
@Saket mentioned this in the ticket, maybe we can confirm with him once.

Reply:
Think you tagged the wrong Saket 🐿

Reply:
classic Saket mix-up! sorry about that.